### PR TITLE
Full release of actions/core 1.6.0 with oidc behavior

### PR DESCRIPTION
**Pull request by thboop**

### CODE REVIEW

File: packages/core/RELEASES.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # @actions/core Releases
 
+### 1.6.0
+- [Added OIDC Client function `getIDToken`](https://github.com/actions/toolkit/pull/887)
```
Comment (TingluoHuang) [inline packages/core/RELEASES.md:4]: we need to update the link

File: packages/core/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@actions/core",
-  "version": "1.5.0",
+  "version": "1.6.0-beta.0",
```
Comment (TingluoHuang) [inline packages/core/package.json:3]: need to update the version.

File: packages/core/__tests__/core.test.ts [hunk 3]
```diff
@@ -434,3 +435,20 @@ function verifyFileCommand(command: string, expectedContents: string): void {
     fs.unlinkSync(filePath)
   }
 }
+
+function getTokenEndPoint(): string {
+  return 'https://vstoken.actions.githubusercontent.com/.well-known/openid-configuration'
+}
+
+describe('oidc-client-tests', () => {
```
Comment (TingluoHuang) [inline packages/core/__tests__/core.test.ts:443]: I hope we could have some real tests.

**Review by luketomlinson — APPROVED** [review]
:shipit:

**Review by TingluoHuang — APPROVED** [review]

File: packages/core/src/oidc-utils.ts [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+    return new HttpClient(
+      'actions/oidc-client',
+      [new BearerCredentialHandler(OidcClient.getRequestToken())],
+      requestOptions
+    )
+  }
+
+  private static getRequestToken(): string {
+    const token = process.env['ACTIONS_ID_TOKEN_REQUEST_TOKEN']
```
Comment (jiasli) [inline packages/core/src/oidc-utils.ts:29]: Using `ACTIONS_ID_TOKEN_REQUEST_TOKEN` is inconsistent with the document where it suggests using `ACTIONS_RUNTIME_TOKEN`:

https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-cloud-providers#requesting-the-jwt-using-environment-variables

```yaml
jobs:
  job:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/github-script@v6
      id: script
      timeout-minutes: 10
      with:
        debug: true
        script: |
          const token = process.env['ACTIONS_RUNTIME_TOKEN']
          const runtimeUrl = process.env['ACTIONS_ID_TOKEN_REQUEST_URL']
          core.setOutput('TOKEN', token.trim())
          core.setOutput('IDTOKENURL', runtimeUrl.trim())
```

Also see https://github.com/github/docs/issues/32573

**Review by Redmishiaomi — APPROVED** [review]

**Pull request merged**